nncf/onnx/graph/nncf_graph_builder.py

```diff
@@ -176,10 +176,13 @@ def _get_node_attrs(node: onnx.NodeProto, model: onnx.ModelProto) -> Dict[str, A
     :param model: ONNX model.
     :return : Node attributes.
     """
+    # We need to collect list of the inputs.
```

Comment:
I think you can safely remove these comments or put them in the docstring in a way that describes the content of attribute dict.